### ISSUE TITLE
LazyVim integration guide not existing

### ISSUE THREAD

**Issue by gaardhus** [quarto-dev/quarto-nvim#133]
Trying to integrate quarto-nvim with LazyVim, I was happy to see a link to a guide to do exactly that in the readme: https://github.com/jmbuhr/lazyvim-starter-for-quarto/blob/main/lua/plugins/quarto.lua

However, the link is dead, and I cannot find it through search.

**Comment by jmbuhr** [comment]
Thanks for pointing that out. I have indeed made that repository private, as it was too much work to maintain consistency with some else's config. I found I can only do that for one config that I'm also daily driving myself to catch interactions with other plugins early. For now I can thus only refer you to https://github.com/jmbuhr/quarto-nvim-kickstarter/

I am working on some changes to `otter.nvim`, though that reduce the overall need for configuring it due to plugin interactions (https://github.com/jmbuhr/otter.nvim/pull/137)
